client/verta/verta/_registry/model.py

```diff
@@ -84,3 +84,33 @@ def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=Non
 
         print("created new RegisteredModel: {} in {}".format(registered_model.name, WORKSPACE_PRINT_MSG))
         return registered_model
+
+    def _get_existing(self):
+        registered_model = RegisteredModel(self._conn, self._conf, self._get_proto_by_id(self._conn, self.id))
+        return registered_model
```

Comment:
We should not add this function.
```suggestion
```

Instead of
```python
registered_model = self._get_existing()
registered_model._msg.labels.append(label)
```
let's use
```python
self._refresh_cache()
self._msg.labels.append(label)
```
etc.

Reply:
replaced with clear refresh cache